simulation/README.md

```diff
@@ -44,8 +42,8 @@ Make sure you have some version of python installed on your personal computer. *
 
 ## Running the Simulation
 ### Docker Simulation Setup
-1. Open up a terminal and ssh into your Docker Container with the `-XY` flag (i.e. run `ssh -XY -p 2200 duke@[ip address]`).
-2. Run `source /opt/ros/kinetic/setup.bash` and `roscore &`.
+1. Open up a terminal and ssh into your Docker Container with the `-XY` flag (i.e. run `ssh -XY -p 2201 duke@[ip address]`).
+2. Run `source /opt/ros/melodic/setup.bash` and `roscore &`.
 3. Run `cd dev/robosub-ros/simulation` and then `./runSim.sh &` (you may first need to run `chmod +x runSim.sh`). Wait until the terminal says `Initialization successful.` If it delays on the video compression library or meshcalc for an extended period of time, press enter a couple of times. This may be nothing more than confirmation-bias superstition.
 4. Run whatever ROS topic publishing code you have. In `robosub-ros/simulation/docker`, there is a python script `squareCommand.py` that you can run to make the robot move approximately in a square. (In reality, the robot will spin in wide circles because the robot isn't balanced.) You can use this script to test if communication between the simulations is working.
 
```

Comment:
Beneath this line, there is discussion about windows 10 pro and docker toolbox. Delete all of that.

Reply:
Deleted.